### PR TITLE
`26.0.1`, `25.0.6` and `24.0.12`

### PR DESCRIPTION
**Pull request by skjnldsv**
For releases tomorrow:
- [x] Move releases from `prereleases` to `releases`
- [x] Mark 24 as EOL

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -110,98 +110,98 @@
...
 			],
 		],
 		'25' => [
 			'100' => [
-				'latest' => '25.0.5',
-				'internalVersion' => '25.0.5.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-25.0.5.zip',
+				'latest' => '25.0.6',
+				'internalVersion' => '25.0.6.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-25.0.6.zip',
```
Comment (nickvergessen) [inline config/config.php:147]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-25.0.6.zip',
```

File: config/config.php [hunk 2]
```diff
@@ -110,98 +110,98 @@
...
 		],
-		'25.0.5' => [
+		'25.0.6' => [
 			'70' => [
-				'latest' => '26.0.0',
-				'internalVersion' => '26.0.0.11',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.0.zip',
+				'latest' => '26.0.1',
+				'internalVersion' => '26.0.1.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.1.zip',
```
Comment (nickvergessen) [inline config/config.php:131]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.1.zip',
```

File: config/config.php [hunk 3]
```diff
@@ -110,98 +110,98 @@
...
 		],
-		'24.0.11' => [
+		'24.0.12' => [
 			'100' => [
-				'latest' => '25.0.5',
-				'internalVersion' => '25.0.5.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-25.0.5.zip',
+				'latest' => '25.0.6',
+				'internalVersion' => '25.0.6.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-25.0.6.zip',
```
Comment (nickvergessen) [inline config/config.php:163]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-25.0.6.zip',
```

File: config/config.php [hunk 4]
```diff
@@ -110,98 +110,98 @@
...
 			],
 		],
 		'24' => [
 			'100' => [
-				'latest' => '24.0.11',
-				'internalVersion' => '24.0.11.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-24.0.11.zip',
+				'latest' => '24.0.12',
+				'internalVersion' => '24.0.12.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-24.0.12.zip',
```
Comment (nickvergessen) [inline config/config.php:179]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-24.0.12.zip',
```

File: config/config.php [hunk 5]
```diff
@@ -110,98 +110,98 @@
...
 			],
 		],
 		'23.0.12' => [
 			'100' => [
-				'latest' => '24.0.11',
-				'internalVersion' => '24.0.11.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-24.0.11.zip',
+				'latest' => '24.0.12',
+				'internalVersion' => '24.0.12.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-24.0.12.zip',
```
Comment (nickvergessen) [inline config/config.php:195]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-24.0.12.zip',
```

File: config/config.php [hunk 6]
```diff
@@ -635,101 +635,101 @@
 	'beta' => [
 		'26' => [
 			'100' => [
-				'latest' => '26.0.1 RC1',
-				'internalVersion' => '26.0.1.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.1rc1.zip',
+				'latest' => '26.0.1',
+				'internalVersion' => '26.0.1.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.1.zip',
```
Comment (nickvergessen) [inline config/config.php:640]: Same here and further

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: config/config.php [hunk 7]
```diff
@@ -110,98 +110,98 @@
...
+pBrUTcbtOHpa45Y+hma2QXFCu5CeijX29314/9y5fLUNwTSbLyxd3JCN+XMEOz2c
+DWIgtNvJa65/puWL3Fsj/ECaKoshnNhhdxS4Zau2fC17HgZNhbM1YvIRHRbvErsV
+4avngST5XyLyaPWe/KI8GdQVZhVuDpmTa/XNQQ1y3esJkD8i9HaLM8A9/pzXTqv2
+B2l/MzsxiGbu8KDp7upj3+uqNatEmS7VvA+a1Afuz+gNrJCx/KbZV33RFhQNAqwm
+YChgDMK2mgiJ2mjb3Dvv4Q==',
 			],
 		],
-		'25.0.5' => [
+		'25.0.6' => [
 			'70' => [
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply config/config.php:128]: Thanks for the heads up! :+1:

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**